src/DynamoCoreWpf/Views/FileTrust/FileTrustWarning.xaml.cs

```diff
@@ -135,9 +135,10 @@ private void YesButton_Click(object sender, RoutedEventArgs e)
                 if (dynViewModel.PreferenceSettings.AddTrustedLocation(fileTrustWarningViewModel.DynFileDirectoryName))
                     dynViewModel.MainGuideManager.CreateRealTimeInfoWindow(string.Format(Properties.Resources.TrustLocationAddedNotification, fileTrustWarningViewModel.DynFileDirectoryName));
             }
-            if (dynViewModel.CurrentSpaceViewModel.RunSettingsViewModel.Model.RunType != RunType.Manual)
+            // The maximum workspaces always will be Two, The first one for the Dynamo file and the second one for a Custom Node
+            if (dynViewModel.Workspaces[0].RunSettingsViewModel.Model.RunType != RunType.Manual)
```

Comment:
I think it's safer to not assume this won't change, and instead check that the workspace is a home workspace.

Reply:
I agree